cypress/support/fragments/check-out-actions/check-out-actions.js

```diff
@@ -12,7 +12,17 @@ export default {
     cy.do([
       TextField({ name: 'item.barcode' }).fillIn(itemBarcode),
       Button({ id: 'clickable-add-item' }).click(),
-      Modal().find(Button('Check out')).click(),
+    ]);
+    if (cy.do(Heading('Item not checked out').exists())) {
+      cy.do([
+        Button('Override').click(),
+        Form({ id: 'override-form' })
+          .find(TextArea({ id: 'textarea-input-10' }))
```

Comment:
think to use something more special instead of id with index

Reply:
may be textarea in the modal form? There is the only one textarea there